Gems/SimulationInterfaces/Code/Include/SimulationInterfaces/SimulationInterfacesBus.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <SimulationInterfaces/SimulationInterfacesTypeIds.h>
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/smart_ptr/shared_ptr.h>
+#include <AzFramework/Physics/ShapeConfiguration.h>
+
+namespace SimulationInterfaces
+{
+    //! # A set of filters to apply to entity queries. See GetEntities, GetEntitiesStates.
+    //! # The filters are combined with a logical AND.
+    //! context : https://github.com/ros-simulation/simulation_interfaces/blob/main/msg/EntityFilters.msg
```

Comment:
consider using doxygen tags e.g.:
```suggestion
    //! @see <a href="https://github.com/ros-simulation/simulation_interfaces/blob/main/msg/EntityFilters.msg">EntityFilters</a>
```